data/pathfinder/paizo/roleplaying_game/advanced_class_guide/acg_abilities_class.lst

```diff
@@ -311,7 +316,7 @@ Bloodrager Bloodline ~ Abyssal Bonus Spells	CATEGORY:Special Ability	TYPE:Intern
 Bloodrager Bloodline ~ Arcane Bonus Spells	CATEGORY:Special Ability	TYPE:Internal	VISIBLE:NO	SPELLKNOWN:CLASS|Bloodrager=1|Magic Missile|PREVARGTEQ:BloodlineCasterLVL,7	SPELLKNOWN:CLASS|Bloodrager=2|Invisibility|PREVARGTEQ:BloodlineCasterLVL,10	SPELLKNOWN:CLASS|Bloodrager=3|Lightning Bolt|PREVARGTEQ:BloodlineCasterLVL,13	SPELLKNOWN:CLASS|Bloodrager=4|Dimension Door|PREVARGTEQ:BloodlineCasterLVL,16
 Bloodrager Bloodline ~ Celestial Bonus Spells	CATEGORY:Special Ability	TYPE:Internal	VISIBLE:NO	SPELLKNOWN:CLASS|Bloodrager=1|Bless|PREVARGTEQ:BloodlineCasterLVL,7	SPELLKNOWN:CLASS|Bloodrager=2|Resist Energy|PREVARGTEQ:BloodlineCasterLVL,10	SPELLKNOWN:CLASS|Bloodrager=3|Heroism|PREVARGTEQ:BloodlineCasterLVL,13	SPELLKNOWN:CLASS|Bloodrager=4|Holy Smite|PREVARGTEQ:BloodlineCasterLVL,16
 Bloodrager Bloodline ~ Destined Bonus Spells	CATEGORY:Special Ability	TYPE:Internal	VISIBLE:NO	SPELLKNOWN:CLASS|Bloodrager=1|Shield|PREVARGTEQ:BloodlineCasterLVL,7	SPELLKNOWN:CLASS|Bloodrager=2|Blur|PREVARGTEQ:BloodlineCasterLVL,10	SPELLKNOWN:CLASS|Bloodrager=3|Protection from Energy|PREVARGTEQ:BloodlineCasterLVL,13	SPELLKNOWN:CLASS|Bloodrager=4|Freedom of Movement|PREVARGTEQ:BloodlineCasterLVL,16
-Bloodrager Bloodline ~ Draconic Bonus Spells	CATEGORY:Special Ability	TYPE:Internal	VISIBLE:NO	SPELLKNOWN:CLASS|Bloodrager=1|Shield|PREVARGTEQ:BloodlineCasterLVL,7	SPELLKNOWN:CLASS|Bloodrager=2|Resist Energy|PREVARGTEQ:BloodlineCasterLVL,10	SPELLKNOWN:CLASS|Bloodrager=3|Fly|PREVARGTEQ:BloodlineCasterLVL,13	SPELLKNOWN:CLASS|Bloodrager=4|Fear|PREVARGTEQ:BloodlineCasterLVL,16
+Bloodrager Bloodline ~ Draconic Bonus Spells	CATEGORY:Special Ability	TYPE:Internal	VISIBLE:NO	SPELLKNOWN:CLASS|Bloodrager=1|Shield|PREVARGTEQ:BloodlineCasterLVL,7|PRESPELLTYPE:1,Arcane=1	SPELLKNOWN:CLASS|Bloodrager=2|Resist Energy|PREVARGTEQ:BloodlineCasterLVL,10|PRESPELLTYPE:1,Arcane=2	SPELLKNOWN:CLASS|Bloodrager=3|Fly|PREVARGTEQ:BloodlineCasterLVL,13|PRESPELLTYPE:1,Arcane=3	SPELLKNOWN:CLASS|Bloodrager=4|Fear|PREVARGTEQ:BloodlineCasterLVL,16|PRESPELLTYPE:1,Arcane=4
```

Comment:
I did this both for the sorcerer and the bloodrager. If you have a sorcerer5/dragon disciple6, you have only level 4 spell slots, but your bloodline is advanced enough to grant you the 5th level bonus spell, so it shows up on your character sheet even if you can't cast it. The Dragon Disciple class description even mentions this case. With this you only get the bonus spells added to your spells known list if you have a spellslot of the appropriate level.